<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>detectorMain</class>
 <widget class="QMainWindow" name="detectorMain">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>detectorMain</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="chooseInputDeviceButton">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>10</y>
      <width>141</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Choose Input Device</string>
    </property>
   </widget>
   <widget class="QPushButton" name="beginRecordingButton">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>70</y>
      <width>141</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Begin Recording</string>
    </property>
   </widget>
   <widget class="QLabel" name="noteLabel">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>130</y>
      <width>451</width>
      <height>331</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>48</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Note Played</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="stopRecordingButton">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>70</y>
      <width>141</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Stop Recording</string>
    </property>
   </widget>
   <widget class="QComboBox" name="inputDevicesComboBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>451</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QCheckBox" name="generateMidiCheckBox">
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>80</y>
      <width>111</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Generate Midi</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="outputFileName">
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>120</y>
      <width>181</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
